json/test_dawjsonlink.cpp

```diff
@@ -92,7 +92,7 @@ coordinate_t calc(const string& text) {
   auto x = 0.0, y = 0.0, z = 0.0;
   auto len = 0;
 
-  using range_t = daw::json::json_array_range<coordinate_t, daw::json::NoCommentSkippingPolicyUnchecked>;
```

Comment:
Other tests may have tweaks like that too. If that is really a game changer (i.e. the time diff > 10%), I'd recommend to have 2 tests based on that code: one is unchecked, and another is checked (could be controlled either by CLI arguments, or compile-time directives).

Reply:
Maybe, but very unlikely this can be in the other libs. I have reviewed the top libraries. Rust is OK, simdjson is OK. This kind of feature is never the default for adequate API. Asdf had a similar feature too, but I have removed it because this is a feature for benchmarks, and very rarely used in real life.

Reply:
What this checked means? Is this utf-8 validation or check correct json format, or what?

Reply:
It's a bit complicated but in a vast majority of cases it is end of input checks, in other cases it may be JSON correctness too.